pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ oraclejdk, stdenv, fetchurl, makeWrapper }:
```

Comment:
is oraclejdk really necessary or does any jre do the job ? if this is the case, you can replace this by `jre` I think.

Reply:
Thank you for your suggestion. Formerly, `oraclejdk` was indeed required here; after your comment I tried out `jre` once more—and it does seem to work now as well.